endpoints/publicProxy/http.go

```diff
@@ -253,6 +253,30 @@ func authHandler(handler http.Handler, pcfg *Config, key []byte, ctx ziti.Contex
 										oauthLoginRequired(w, r, pcfg.Oauth, provider.(string), target, authCheckInterval)
 										return
 									}
+
+									// Add check for 'aud' claim
+									var requestedHost string
+									if host := r.Header.Get("Host"); host != "" {
+										requestedHost = host
+									} else if xForwardedHost := r.Header.Get("X-Forwarded-Host"); xForwardedHost != "" {
+										requestedHost = xForwardedHost
+									}
+
+									if requestedHost == "" {
+										logrus.Debug("all checked headers (Host, X-Forwarded-Host) are missing. Here are the present headers:")
+										for name, values := range r.Header {
```

Comment:
i'd remove all this and maybe replace the log message and just indicate neitehr host nor x-f-host were found. if that's the case, is this a "bigger deal"? should it be an error?